core/src/main/java/com/datastrato/gravitino/listener/event/CreateTableEvent.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.listener.event;
+
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.annotation.DeveloperApi;
+import com.datastrato.gravitino.rel.Table;
+
+/**
+ * Represents an event triggered after a table is successfully created. This class extends {@link
+ * TableEvent} to encapsulate specific details related to the creation of a table, including both
+ * the request used to initiate the table creation and the resulting table information provided to
+ * the user upon successful creation.
+ *
+ * <p>This event can be useful for auditing operations, tracking table creation activities, or
+ * understanding the configuration and state of a newly created table.
+ */
+@DeveloperApi
+public class CreateTableEvent extends TableEvent {
+  private Table createdTableInfo;
+  private Table createTableRequest;
```

Comment:
It's a little strange to also define the `createTableRequest` as a `Table`.

1) Do we need to expose the request?
2) If so, is there any better definition to describe this?

Reply:
remove `createTableRequest` from `CreateTableEvent`, but leaves it in `CreateTableFailureEvent`